tests/issues/test_88_random_error.py

```diff
@@ -28,8 +28,7 @@ async def test_notification_validation_error(tmp_path: Path):
 
     server = Server(name="test")
     request_count = 0
-    slow_request_started = anyio.Event()
-    slow_request_complete = anyio.Event()
+    slow_request = anyio.Event()
```

Comment:
nit: `slow_request_lock` to make it obvious what this is when reading the test further down